lib/src/Makefile.am

```diff
@@ -1,4 +1,4 @@
-nobase_include_HEADERS = ovis-test/test.h ovis-lib-config.h
```

Comment:
I think `ovis-lib-config.h` is generated by `configure` script and should not be distributed. I think adding `ovis-test/test.h` to the `EXTRA_DIST` list is more preferable.